FormApproaching/PhotoPicker/PhotoPickerViewModelBuilder.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+protocol PhotoPickerViewModelBuilding {
+    func buildViewModel(updates: PhotoPickerTableViewUpdates) -> PhotoPickerTableViewModel
+}
+
+enum PhotoPickerViewCoordination {
+    case selected(URL)
+}
+
+struct PhotoPickerTableViewModelBuilder: PhotoPickerViewModelBuilding {
+    private let photoAlbumProvider: PhotoAlbumProviding
+    private let coordination: (PhotoPickerViewCoordination) -> Void
+    private let cancellableStore = CancellableStore()
+
+    init(
+        photoAlbumProvider: PhotoAlbumProviding,
+        coordination: @escaping (PhotoPickerViewCoordination) -> Void
+    ) {
+        self.photoAlbumProvider = photoAlbumProvider
+        self.coordination = coordination
+    }
+    
+    func buildViewModel(updates: PhotoPickerTableViewUpdates)
+        -> PhotoPickerTableViewModel {
+            photoAlbumProvider.getPhotos { result in
```

Comment:
`updates` should be captured weakly / unowned

Reply:
but this is a structure